distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/MachineDependentHeap.java

```diff
@@ -140,11 +140,13 @@ enum MachineNodeRole {
          *     <li>The absolute maximum heap size is 31 gigabytes.</li>
          * </ul>
          *
+         * In all cases the result is rounded down to the next whole multiple of 4 megabytes.
```

Comment:
Should we add here an explanation for why we're rounding down to the next multiple of 4 MB?